Tests/AppTests/PackageShowModelTests.swift

```diff
@@ -45,6 +46,35 @@ class PackageShowModelTests: SnapshotTestCase {
         XCTAssertEqual(m?.title, "bar")
     }
 
+    func test_init_withSPIManifest() async throws {
+        let pkg = try savePackage(on: app.db, "1".url)
+        try await Repository(package: pkg, name: "bar", owner: "foo").save(on: app.db)
+        let version = try App.Version(package: pkg,
+                                      commit: "12345",
+                                      commitDate: Current.date(),
+                                      latest: .defaultBranch,
+                                      packageName: "Package",
+                                      reference: .branch("main"),
+                                      spiManifest: try Manifest(yml:
+                                         """
+                                         version: 1
+                                         builder:
+                                           configs:
+                                           - documentation_targets: [Target1, Target2]
+                                         """))
```

Comment:
This test feels like it's also re-testing the yml parsing of the Manifest, but it was just the quickest and easiest way to get targets into a config.

Reply:
Yeah, I don't think there's much merit to this test. The bit that's relevant - whether the value is properly assigned in the `init` - is tested at a high level in the snapshot test.

The purpose of the `test_init_no_packageName` above was to test a pathological case, which is different.

I'd say we drop it.

Reply:
Removed.